### PR TITLE
LPS-166868 | master

### PR DESCRIPTION
**Pull request by sontruongces**
Hi team,

This is the fix for the [LPS-166868](https://issues.liferay.com/browse/LPS-166868), the original has passed my review at sontruongces#61.

**Step to reproduces:** [LPS-166868](https://issues.liferay.com/browse/LPS-166868)
**Discussion:** https://liferay.slack.com/archives/CLD39UKM5/p1668571278941749

Thanks!
cc: @tototrinh

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/document-library-opener/document-library-opener-google-drive-web/src/main/java/com/liferay/document/library/opener/google/drive/web/internal/DLOpenerGoogleDriveManager.java [hunk 1]
```diff
@@ -145,6 +148,21 @@ public void delete(long userId, FileEntry fileEntry)
...
+					deleteDLOpenerFileEntryReference(
+						DLOpenerGoogleDriveConstants.
+							GOOGLE_DRIVE_REFERENCE_TYPE,
+						fileEntry);
+
+				return;
+			}
+
 			throw new PortalException(ioException);
 		}
```
Comment (adolfopa) [inline modules/apps/document-library-opener/document-library-opener-google-drive-web/src/main/java/com/liferay/document/library/opener/google/drive/web/internal/DLOpenerGoogleDriveManager.java:167]: Can you split this in two catch blocks?
```
catch (GoogleJsonResponseException ...) {
    if (_log.isDebugEnabled()) {
        ...
     }

    ...
}
catch (IOException ioException) {
    throw new PortalException(...);
}
```

File: modules/apps/document-library-opener/document-library-opener-google-drive-web/src/main/java/com/liferay/document/library/opener/google/drive/web/internal/DLOpenerGoogleDriveManager.java [hunk 2]
```diff
@@ -333,6 +333,15 @@ private File _getContentFile(long userId, FileEntry fileEntry) {
...
 		catch (IOException | PortalException exception) {
+			if (exception instanceof GoogleJsonResponseException) {
+				if (_log.isDebugEnabled()) {
+					_log.debug(
+						"The GoogleDrive file does not exist", exception);
+				}
+
+				return null;
+			}
+
```
Comment (adolfopa) [inline modules/apps/document-library-opener/document-library-opener-google-drive-web/src/main/java/com/liferay/document/library/opener/google/drive/web/internal/DLOpenerGoogleDriveManager.java:385]: Same as before; if possible, split in two catch clauses.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by tototrinh** [comment]
Hi @adolfopa thanks for your feedback. I updated the code, please take a look again. Thanks.

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**